### ISSUE TITLE
Build and release error

### ISSUE THREAD

**Issue by basilesimon** [beabee-communityrm/beabee-frontend#163]
See https://github.com/beabee-communityrm/beabee-frontend/runs/6095088651?check_suite_focus=true

```
#18 [builder 6/6] RUN npx vite build
#18 sha256:707e4905b3a601d3758d5df60f625163742d5b9c6bd4edc9f0a6e72fb45e78c0
#18 1.221 vite v2.6.14 building for production...
#18 1.288 transforming...
#18 1.726 ✓ 10 modules transformed.
#18 1.726 [vite-plugin-vue-i18n] Cannot read properties of undefined (reading 'message')
#18 1.726 file: /opt/beabee-frontend/locales/de.json
#18 1.727 error during build:
#18 1.727 TypeError: Cannot read properties of undefined (reading 'message')
#18 1.727     at Object.newOptions.onError (/opt/beabee-frontend/node_modules/@intlify/bundle-utils/lib/codegen.js:102:48)
#18 1.727     at Object.newOptions.onError (/opt/beabee-frontend/node_modules/@intlify/bundle-utils/lib/codegen.js:102:36)
#18 1.727     at newOptions.onError (/opt/beabee-frontend/node_modules/@intlify/bundle-utils/lib/codegen.js:102:36)
#18 1.727     at emitError (/opt/beabee-frontend/node_modules/@intlify/message-compiler/dist/message-compiler.cjs.prod.js:174:13)
#18 1.727     at readTokenInLinked (/opt/beabee-frontend/node_modules/@intlify/message-compiler/dist/message-compiler.cjs.prod.js:728:21)
#18 1.727     at readToken (/opt/beabee-frontend/node_modules/@intlify/message-compiler/dist/message-compiler.cjs.prod.js:742:20)
#18 1.727     at Object.nextToken (/opt/beabee-frontend/node_modules/@intlify/message-compiler/dist/message-compiler.cjs.prod.js:784:16)
#18 1.727     at parseLinked (/opt/beabee-frontend/node_modules/@intlify/message-compiler/dist/message-compiler.cjs.prod.js:919:31)
#18 1.727     at parseMessage (/opt/beabee-frontend/node_modules/@intlify/message-compiler/dist/message-compiler.cjs.prod.js:1018:36)
#18 1.727     at parseResource (/opt/beabee-frontend/node_modules/@intlify/message-compiler/dist/message-compiler.cjs.prod.js:1057:25)
#18 ERROR: executor failed running [/bin/sh -c npx vite build]: exit code: 1
------
 > [builder 6/6] RUN npx vite build:
------
executor failed running [/bin/sh -c npx vite build]: exit code: 1
Service 'frontend' failed to build
```

**Comment by basilesimon** [comment]
`/src/modules/admin/AdminPage.vue`

```js
    <Hint class="content-i18n" v-html="t('adminDashboard.hint1')" />
    <Hint class="content-i18n" v-html="t('adminDashboard.hint2')" />
    <Hint class="content-i18n" v-html="t('adminDashboard.hint3')" />
  </div>
  <p class="text-center text-sm mt-8">
    {{ t('adminDashboard.supportInbox') }}
    <a class="text-link" href="mailto:support@beabee.io">support@beabee.io</a>
  </p>
```

**Comment by wpf500** [comment]
Fixed by #164
